python/tskit/drawing.py

```diff
@@ -683,7 +693,26 @@ def draw(self):
 
             # Add node symbol + label next (visually above the edge subtending this node)
             # Symbols
-            curr_svg_group.add(dwg.circle(**self.node_attrs[u]))
+            if self.tree.is_sample(u):
+                curr_svg_group.add(
+                    dwg.rect(
+                        **{
```

Comment:
This a lot of vspace. Why not create the args dict in a variable rather than doing everything in one call?

Reply:
Yeah, it was stuck in by black. I can use a variable.